package.json

```diff
@@ -34,37 +32,39 @@
     "@typescript-eslint/eslint-plugin": "^5.62.0",
     "@typescript-eslint/parser": "^5.61.0",
     "chai": "^4.3.10",
-    "eslint": "^8.51.0",
+    "eslint": "^8.52.0",
     "eslint-config-prettier": "^8.10.0",
     "eslint-config-salesforce": "^2.0.2",
     "eslint-config-salesforce-license": "^0.2.0",
-    "eslint-config-salesforce-typescript": "^1.1.2",
+    "eslint-config-salesforce-typescript": "^2.0.0",
     "eslint-plugin-header": "^3.1.1",
     "eslint-plugin-import": "2.28.1",
-    "eslint-plugin-jsdoc": "^43.0.5",
+    "eslint-plugin-jsdoc": "^46.8.2",
     "eslint-plugin-sf-plugin": "^1.16.9",
     "husky": "^7.0.4",
     "mocha": "^9.1.3",
     "nyc": "^15.1.0",
-    "oclif": "^3.17.2",
+    "oclif": "^4.0.3",
     "prettier": "^2.8.8",
     "pretty-quick": "^3.1.0",
     "shx": "0.3.4",
     "sinon": "10.0.0",
     "sinon-chai": "^3.7.0",
-    "ts-node": "^10.0.0",
-    "typescript": "^4.9.5",
-    "wireit": "^0.14.0"
+    "ts-node": "^10.9.1",
+    "typescript": "^5.2.2",
+    "wireit": "^0.14.1"
   },
   "config": {},
   "engines": {
-    "node": ">=16.0.0"
+    "node": ">=18.0.0"
   },
   "files": [
     "bin",
     "/lib",
     "/messages",
-    "/oclif.manifest.json"
+    "/oclif.manifest.json",
+    "/oclif.lock",
+    "/oclif.lock"
```

Comment:
only one

Reply:
related to oclif.lock - the `postpack` and `clean:lib` scripts should remove the `oclif.lock` too... Probably should add that to our migrate command